src/pathfinding_service/cli.py

```diff
@@ -126,7 +126,7 @@ def main(  # pylint: disable=too-many-arguments,too-many-locals
         service.start()
         log.debug("Waiting for service to start before accepting API requests")
         try:
-            service.startup_finished.get(timeout=60)
+            service.startup_finished.get(timeout=PFS_START_TIMEOUT)
```

Comment:
Should we adapt the for the MSRC as well?

Reply:
I only found the `MATRIX_START_TIMEOUT` in the MSRC and not additional timeout like in the PFS.